CogX abstract: turing way + culture of collaboration (#352)
WhitakerLab/WhitakerLabProjectManagement: talk-proposals/CogX-TuringWay-MS.md

## Talk Description

_Please give a brief description of your talk (max 500 words)._
_The evaluation criteria of proposals will be:_

- _Expected general interest of the talk_
- _Academic standard and global impact_
- _Breadth of domain application_

Online behaviour over the last months during the COVID-19 pandemic has shifted significantly. 
With buzzwords like “flatten the curve” and trending hashtags on Twitter like “#StayHomeSaveLives”, we have become more inclined to keep abreast of what exactly is going on in our communities. 
Digital technologies and approaches are playing vital roles in creating opportunities for us to take meaningful actions as leaders, researchers, innovators and community members.
Already 100s of online projects have been launched to promote data-driven solutions that can take on medical and social challenges related to coronavirus pandemic. 
Individuals, who want to lead and participate in such projects, are collaborating to identify interdisciplinary applications of artificial intelligence and machine learning.

There has never been a better time for online platforms and communities to thrive and test innovative ideas for a common social cause.

> However, participation in such projects also requires an understanding of project design, data management, software development, leadership and collaboration techniques: skills that are not widely taught to academic leaders and data scientists.

Author: ```suggestion
However, participation in such projects will also demand an understanding of project design, data management, software development, leadership and collaboration techniques: skills that are not widely taught to academic leaders and data scientists.
```

Digital technologies and data-driven approaches have played vital roles during this situation in creating opportunities for us to take meaningful actions as leaders, researchers, innvovators and community members.
Already 100s of online projects have been launched to promote the development of technical solutions that can take on medical and social challenges related to coronavirus pandemic. 
There has never been a better time for online platforms and communities to thrive and test innovative ideas for a common social cause.
This is particularly empowering for individuals who want to lead and involve others in data-driven projects that identify interdisciplinary applications of artificial intelligence (AI) and machine learning (ML).
However, such efforts require understanding of project design, data management, software development, and continuous integration techniques: skills that are not widely taught to academic leaders and data scientists.

In this talk, I will focus on the importance of documenting and sharing knowledge around best practices in data science that can be adopted by various stakeholder in the current crisis and in the future.

> Such resources, when collaboratively developed, can have a significant impact on improving access to information and the decision-making process by our peers and future generations.

Reviewer: Add a sentence here that says something like "projects and initiatives must be designed for inclusion from the start"

Aligned with values of CogX, The Turing Way promotes the inclusion of diverse stakeholders by creating an open community-led platform that invites them to collaborate and work with each other.

With the help of The Turing Way, I would like to discuss the importance of creating a culture that promotes collaboration to ensure the long-term sustainability of our communities.
In data science projects, it is required to establish common standards for software, research methods, and dataset for all the collaborators.
However, it is equally important that the leaders invest their resources in designing projects that are truly inclusive and supportive of their members.
This means that everyone involved in a project or research is given a fair opportunity to 1) develop a common technical understanding of the project, 2) state their expectations from the collaboration, 3) access the resources required in the project, and 4) advance their skills to contribute equitably.
It is a very challenging process, especially because these skills are not formally taught to everyone in academia and data research.

> The Turing Way aims to fill this gap in skills for members interested in data science and leaders who want to create projects that accessible for their community and team members.

Author: ```suggestion
The Turing Way aims to fill this gap in skills for members interested in data science and leaders who want to create projects that are accessible for their community and team members.
```

The Turing Way is a community-led book project that offers open-source learning and teaching resources that help researchers enhance their skills.
As a live book, it also provides a platform that brings different stakeholders together to read, write, edit, and update its chapters in real-time and collaborate over the internet.
Contributors are encouraged to bring their perspective into the project, enhance the existing resources or create new ones, and get fairly acknowledged for their work.

> The Turing Way welcomes ideas, case studies, impact stories, educational resources, or any sort of participation that individuals can benefit from or advance their knowledge in data research.

Author: ```suggestion
The Turing Way welcomes ideas, case studies, impact stories, educational resources, or any sort of participation that individuals can benefit from or use to advance their knowledge in data research.
```

- _Expected general interest of the talk_
- _Academic standard and global impact_
- _Breadth of domain application_

Online behaviour over the last months during the COVID-19 pandemic has shifted significantly. 
With buzzwords like “flatten the curve” and trending hashtags on Twitter like “#StayHomeSaveLives”, we have become more inclined to stay informed about what exactly is going on in our communities. 
In the last month alone (in March 2020), 100s of online hackathon events have been launched to promote the development of software that can take on challenges related to coronavirus pandemic. 
There has been no better time for online platforms and communities to thrive and test innovative ideas for a common social cause. 
This is a very empowering time for individuals who want to participate in and lead data-driven projects, and a particularly favourable time for Open Research communities to grow. 
It is, unfortunately, also a time when more people will face barriers to open research communities such as isolation, and exclusion, if not done correctly. 

In this talk, I discuss the importance of designing our communities to be welcoming and accessible.

> I also shed light on how community’s culture, when not intentionally developed by the community organizers and leaders, can create massive barriers in Open Research. 

Author: ```suggestion
I also shed light on how the community’s culture, when not intentionally developed by the community organizers and leaders, can create massive barriers in Open Research. 
```
